steps/reasons-for-appealing/sending-evidence/SendingEvidence.js

```diff
@@ -13,7 +13,7 @@ class SendingEvidence extends Question {
 
     get hasSignedUpForEmail() {
 
-        return this.fields.emailAddress.value !== '';
+        return this.fields.emailAddress.value !== undefined;
```

Comment:
When leaving optional fields blank within one-per-page the `.value` will always be an empty string.